src/components/Facepile/Facepile.html

```diff
@@ -0,0 +1,252 @@
+                  <div class="ms-Persona-presence"></div>
+                  <div class="ms-Persona-details">
+                    <div class="ms-Persona-primaryText">Marcus Lauer</div>
+                    <div class="ms-Persona-secondaryText">Marketing</div>
+                  </div>
+                </div>
+              </button>
+            </li>
+          </ul>
+          <div class="ms-PeoplePicker-searchMore js-searchMore">
+            <button class="ms-PeoplePicker-searchMoreBtn">
+              <div class="ms-PeoplePicker-searchMoreIcon">
+                <i class="ms-Icon ms-Icon--search"></i>
+              </div>
+              <div class="ms-PeoplePicker-searchMorePrimary">Search Contacts &amp; Directory</div>
+            </button>
+          </div>
+        </div>
+        <!-- Members list -->
+        <div class="ms-PeoplePicker-selected is-active">
+          <div class="ms-PeoplePicker-selectedHeader">
+            <span class="ms-PeoplePicker-selectedCount">3</span> member<span>s</span>
+          </div>
+          <ul class="ms-PeoplePicker-selectedPeople">
+            <li class="ms-PeoplePicker-selectedPerson">
+              <div class="ms-Persona ms-Persona--sm">
+                <div class="ms-Persona-imageArea">
+                  <div class="ms-Persona-initials ms-Persona-initials--blue">AL</div>
+                  <img class="ms-Persona-image" src="../persona/Persona.Person2.png">
+                </div>
+                <div class="ms-Persona-presence"></div>
+                <div class="ms-Persona-details">
+                  <div class="ms-Persona-primaryText">Alton Lafferty</div>
+                  <div class="ms-Persona-secondaryText">Accountant</div>
+                </div>
+              </div>
+              <button class="ms-PeoplePicker-resultAction js-selectedRemove"><i class="ms-Icon ms-Icon--x"></i></button>
+            </li>
+            <li class="ms-PeoplePicker-selectedPerson">
+              <div class="ms-Persona ms-Persona--sm">
+                <div class="ms-Persona-imageArea">
+                  <div class="ms-Persona-initials ms-Persona-initials--green">DF</div>
+                  <img class="ms-Persona-image" src="">
+                </div>
+                <div class="ms-Persona-presence"></div>
+                <div class="ms-Persona-details">
+                  <div class="ms-Persona-primaryText">Douglas Fielder</div>
+                  <div class="ms-Persona-secondaryText">Public Relations</div>
+                </div>
+              </div>
+              <button class="ms-PeoplePicker-resultAction js-selectedRemove"><i class="ms-Icon ms-Icon--x"></i></button>
+            </li>
+            <li class="ms-PeoplePicker-selectedPerson">
+              <div class="ms-Persona ms-Persona--sm">
+                <div class="ms-Persona-imageArea">
+                  <div class="ms-Persona-initials ms-Persona-initials--purple">ML</div>
+                  <img class="ms-Persona-image" src="">
```

Comment:
It looks like all these image tags don't have an Alt attribute and also `src=""` cannot be empty, we should have a fallback image at least if we are waiting to load one in.